src/components/ClaimMasterPanel.js

```diff
@@ -448,6 +471,7 @@ class ClaimMasterPanel extends FormPanel {
                     reset={reset}
                     onChange={(v) => this.updateAttribute("explanation", v)}
                     readOnly={ro}
+                    required={this.isExplanationMandatoryForIPD && edited.visitTypeOutInPatient=="IPD" ? true : false}
```

Comment:
Substitute '===' in place of '=='. As in a comment above, it would be great to have a const for 'IPD' with more descriptive name.